src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/HtmlEncoder.cs

```diff
@@ -24,7 +25,7 @@ public static HtmlEncoder Create(params UnicodeRange[] allowedRanges)
         }
     }
 
-    public sealed class DefaultHtmlEncoder : HtmlEncoder
+    internal sealed class DefaultHtmlEncoder : HtmlEncoder
```

Comment:
Of course :smile: 

But the time delta of making-it-public is often significant, even with "rapid" NuGet releases.